modules/global/src/com/haulmont/cuba/core/global/EmailInfo.java

```diff
@@ -38,8 +38,13 @@
 
     /**
      * Recipient email addresses separated with "," or ";" symbol.
+     * flag <pre>sendInOneMessage</pre> created to have backward compatibility with CUBA 6.10.2 version and lower
```

Comment:
1. Fix formatting and wording:
```
<p>
Flag {@code sendInOneMessage} is for backward compatibility with CUBA 6.10.2 version and lower. 
If {@code sendInOneMessage = true} then one message will be sent for all recipients and it will include CC and BCC.
Otherwise CC and BCC are ignored and multiple messages by the number of recipients in addresses will be sent.
```
2. Why since 6.10.2? We usually make changes in data model only in feature releases.

Reply:
Since which CUBA version it can be used, if pull request will be accepted?

Reply:
6.11 or 7.0